docs: clarify process to connect to 3/3X over ADB (#35090)
commaai/openpilot: docs/how-to/connect-to-comma.md

## ADB

In order to use ADB on your device, you'll need to perform the following steps:

* Plug in your device to constant power via the OBD-C port
* Enable ADB in your device's settings

> * Plug in your device to your PC using the auxiliary port (the one that comes covered by default)

Reviewer: It doesn't come covered anymore

Reviewer 2: it does, reverted that

Reviewer 2: but this is also a bad way to describe it. just use the picture like we do for the flasher

Author: will add the image and rewrite this paragraph

> * Plug in your device to your PC using the auxiliary port (the one that comes covered by default)

Author: ```suggestion
* Plug in your device to your PC using the lower USB-C port
```
